Sources/WebAuthn/Ceremonies/Registration/PublicKeyCredentialCreationOptions.swift

```diff
@@ -21,6 +21,30 @@ import Foundation
 ///
 /// - SeeAlso: https://www.w3.org/TR/webauthn-2/#dictionary-makecredentialoptions
 public struct PublicKeyCredentialCreationOptions: Encodable, Sendable {
+    
+    init(challenge: [UInt8], 
```

Comment:
Not sure if this was meant to be public, but the internal initializer should be synthesized automatically based on default values on members. If you means to make this public, it would be better in an extension.

Reply:
Also, note that `PublicKeyCredentialCreationOptions` is not intended to be instanciated directly — you are expected to request one from the manager, so many of these options likely belong there instead.

Reply:
Fair point, I'll adjust this one to be in line with your suggestions.

Reply:
Updated in `3a2815e1dd3bdd07bbbb57c311eac272da05bb75`